package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "metronome-contracts",
-  "version": "2.4.0",
+  "version": "2.4.2",
```

Comment:
Don't change the version now. We will publish once this PR is merged and will update the version in a new PR.

Reply:
done